### PR TITLE
Allow `math avg` to work on durations

### PR DESCRIPTION
**Pull request by gorogoroumaru**
issue #2511 

Extend `math avg` to be able to handle durations

Example:
`> echo 1sec 1ms | math avg`
`500ms 500us`

### CODE REVIEW

**Comment by gorogoroumaru** [comment]
I also extend math avg to be able to handle durations.

Example:
`> echo 1sec 1ms | math avg`
`1sec 1ms`

File: crates/nu-cli/src/commands/math/avg.rs [hunk 1]
```diff
@@ -108,6 +125,28 @@ pub fn average(values: &[Value], name: &Tag) -> Result<Value, ShellError> {
                 &name.span,
             )
         })
+    } else if is_duration {
+        to_duration(&sum(
+            UntaggedValue::int(0).into_untagged_value(),
```
Comment (jonathandturner) [inline crates/nu-cli/src/commands/math/avg.rs:130]: Does sum not work over durations directly? If not, I wonder if we could update it to. That way, you wouldn't need to convert in and out of Int.

**Comment by gorogoroumaru** [comment]
Thank you @jonathandturner, I applied your suggested changes.

**Comment by andrasio** [comment]
Looks good!

**Pull request merged**